### PR TITLE
Naive Prometheus import mode support

### PR DESCRIPTION
**Pull request by hagen1778**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,11 +133,151 @@ Here's an example of importing timeseries for one day only:
 
 Please see more about time filtering [here](https://docs.influxdata.com/influxdb/v1.7/query_language/schema_exploration#filter-meta-queries-by-time).
 
-#### Performance
+
+## Migrating data from Prometheus
```
Comment (tenmozes) [inline README.md]: misleading title (prometheus) and help command below (influx)

Comment (hagen1778) [reply README.md]: Hm, I don't see which command below you're pointing to. Could you pls elaborate?

Comment (tenmozes) [reply README.md]: `./vmctl influx --help` line 140

Comment (hagen1778) [reply README.md]: yes, now I see it) thx)

File: prometheus.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+			}
+		}()
+	}
+
+	// any error breaks the import
+	for _, br := range blocks {
+		select {
+		case promErr := <-errCh:
+			return fmt.Errorf("prometheus error: %s", promErr)
+		case vmErr := <-pp.im.Errors():
```
[the remark this answers is not in the document]
Comment (hagen1778) [reply prometheus.go]: How would that improve readability? Could you pls give and example?

File: prometheus.go [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+	"github.com/cheggaaa/pb/v3"
+	"github.com/prometheus/prometheus/tsdb"
+	"github.com/victoriametrics/vmctl/prometheus"
+	"github.com/victoriametrics/vmctl/vm"
+)
+
+type prometheusProcessor struct {
+	cl *prometheus.Client
+	im *vm.Importer
+	cc int
```
[the remark this answers is not in the document]
Comment (hagen1778) [reply prometheus.go]: this is done

File: prometheus.go [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+				bar.Increment()
+			}
+		}()
+	}
+
+	// any error breaks the import
+	for _, br := range blocks {
+		select {
+		case promErr := <-errCh:
+			return fmt.Errorf("prometheus error: %s", promErr)
```
[the remark this answers is not in the document]
Comment (hagen1778) [reply prometheus.go]: made errCh buffered and called close() on dispatcher channel explicitly

File: prometheus.go [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+		case vmErr := <-pp.im.Errors():
+			var errTS string
+			for _, ts := range vmErr.Batch {
+				errTS += fmt.Sprintf("%s for timestamps range %d - %d\n",
+					ts.String(), ts.Timestamps[0], ts.Timestamps[len(ts.Timestamps)-1])
+			}
+			return fmt.Errorf("Import process failed for: \n%swith error: %s", errTS, vmErr.Err)
```
[the remark this answers is not in the document]
Comment (hagen1778) [reply prometheus.go]: made errCh buffered and called close() on dispatcher channel explicitly

File: prometheus.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (hagen1778) [reply prometheus.go]: I don't see much improvement here. I'd be glad if you provide an example. For now I think it is fine.

**Pull request merged**